src/main/java/org/skriptlang/skript/bukkit/whitelist/WhitelistModule.java

```diff
@@ -0,0 +1,25 @@
+package org.skriptlang.skript.bukkit.whitelist;
+
+import java.io.IOException;
+
+import ch.njol.skript.Skript;
+import org.skriptlang.skript.addon.AddonModule;
+import org.skriptlang.skript.addon.SkriptAddon;
+
+public class WhitelistModule implements AddonModule {
+
+	@Override
+	public boolean canLoad(SkriptAddon addon) {
+		return Skript.classExists("com.destroystokyo.paper.event.server.WhitelistToggleEvent");
```

Comment:
this is probably not a great check, since 
a) 1.20.4 will always have this class
b) a lot of the classes do not rely on this event existing.